code/mergeblock/merge.py

```diff
@@ -0,0 +1,74 @@
+#Jakub Podolak for Plastic Monkeys Cansat 2019
+
+import subprocess
+import configparser
+import msgpack
+import time
+
+#our command to get one last line from file
+tail_command = "tail -n 1 "
+
+#initialize config parser, read from config.ini
+config = configparser.ConfigParser()
+config.read('config.ini')
```

Comment:
I should be able to pass different config file like this: `python3 merge.py config7` and then it would read values from file `config7`

Reply:
https://docs.python.org/3/library/sys.html#sys.argv

Reply:
Done - using sys.argv[0]

Reply:
`sys.argv[0]` is a name of the script, we should use `sys.argv[1]` instead
https://docs.python.org/3/library/sys.html#sys.argv

Reply:
Resolved